modules/memo/src/main/Picfit.scala

```diff
@@ -155,6 +243,10 @@ final class PicfitUrl(config: PicfitConfig)(using Executor) extends lila.core.mi
     height = ~size.toOption
   )
 
+  // 560x560 containment consumes the minimum 1601 tokens according to the formula here:
+  // https://docs.together.ai/docs/vision-overview#pricing
+  def forAutomod(id: ImageId) = display(id, "resize")(width = 560, height = 560)
```

Comment:
it results in distorted images, like this: 

<img width="560" height="560" alt="Image" src="https://github.com/user-attachments/assets/42dfc896-54d2-4c12-907f-a20a57b5dc7d" />

Reply:
`display(id, "thumbnail")(width = 560, height = 560)` crops it:

<img width="560" height="560" alt="image" src="https://github.com/user-attachments/assets/342cf737-e20f-4000-ac0e-f247ae715d10" />